sdk-extensions/incubator/src/test/java/io/opentelemetry/sdk/extension/incubator/fileconfig/LogLimitsFactoryTest.java

```diff
@@ -9,40 +9,41 @@
 import static org.mockito.Mockito.mock;
 
 import io.opentelemetry.sdk.autoconfigure.internal.SpiHelper;
+import io.opentelemetry.sdk.extension.incubator.fileconfig.internal.model.AttributeLimits;
 import io.opentelemetry.sdk.extension.incubator.fileconfig.internal.model.LogRecordLimits;
 import io.opentelemetry.sdk.logs.LogLimits;
 import java.util.Collections;
-import org.junit.jupiter.api.Test;
+import java.util.stream.Stream;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
 
 class LogLimitsFactoryTest {
 
-  @Test
-  void create_Null() {
+  @ParameterizedTest
+  @MethodSource("createArguments")
+  void create(LogRecordLimitsAndAttributeLimits model, LogLimits expectedLogLimits) {
     assertThat(
             LogLimitsFactory.getInstance()
-                .create(null, mock(SpiHelper.class), Collections.emptyList()))
-        .isEqualTo(LogLimits.getDefault());
+                .create(model, mock(SpiHelper.class), Collections.emptyList()))
+        .isEqualTo(expectedLogLimits);
   }
 
-  @Test
-  void create_Defaults() {
-    assertThat(
-            LogLimitsFactory.getInstance()
-                .create(new LogRecordLimits(), mock(SpiHelper.class), Collections.emptyList()))
-        .isEqualTo(LogLimits.getDefault());
-  }
-
-  @Test
-  void create() {
-    assertThat(
-            LogLimitsFactory.getInstance()
-                .create(
-                    new LogRecordLimits()
-                        .withAttributeCountLimit(1)
-                        .withAttributeValueLengthLimit(2),
-                    mock(SpiHelper.class),
-                    Collections.emptyList()))
-        .isEqualTo(
-            LogLimits.builder().setMaxNumberOfAttributes(1).setMaxAttributeValueLength(2).build());
+  private static Stream<Arguments> createArguments() {
+    return Stream.of(
+        Arguments.of(null, LogLimits.builder().build()),
+        Arguments.of(
+            LogRecordLimitsAndAttributeLimits.create(new AttributeLimits(), new LogRecordLimits()),
+            LogLimits.builder().build()),
```

Comment:
What if both fields of `LogRecordLimitsAndAttributeLimits` are null?
```suggestion
        Arguments.of(
            LogRecordLimitsAndAttributeLimits.create(null, null),
            LogLimits.builder().build()),
        Arguments.of(
            LogRecordLimitsAndAttributeLimits.create(new AttributeLimits(), new LogRecordLimits()),
            LogLimits.builder().build()),
```
(This applies to all 4 test classes changed here)

Reply:
Good call. Added test coverage.